docs/recipes/upgrading/v4-to-v5.md

```diff
@@ -0,0 +1,257 @@
+# How do I upgrade from SAFE v4 to v5?
+
+1. Upgrade dotnet tools to latest version:
+
+    - `dotnet tool update fable`
+    - `dotnet tool update paket`
+    - `dotnet tool update fantomas`
+    - `dotnet tool update femto`
+
+1. Remove `fantomas-tool` by running `dotnet tool uninstall fantomas-tool`
+    
+1. To upgrade to .NET 8 change the `global.json` to
+
+	```json
+	{
+	  "sdk": {
+	    "version": "8.0.0",
+	    "rollForward": "latestMinor"
+	  }
+	}
+	```
+
+    followed by updating each of your project files to target .NET 8
+
+    ```xml
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+    </PropertyGroup>
+    ```
+
+1. Upgrade .NET dependencies to latest version:
+
+	Update your `paket.dependencies` with:
+		
+	```
+	source https://api.nuget.org/v3/index.json
+	framework: net8.0
+	storage: none
+	
+	nuget Fsharp.Core ~> 8
+
+    //Server
+	nuget Fable.Remoting.Giraffe ~> 5
+	nuget Saturn ~> 0
+	
+    //Client
+	nuget Fable.Core ~> 4
+	nuget Fable.Elmish ~> 4
+	nuget Fable.Elmish.React ~> 4
+	nuget Fable.Elmish.Debugger ~> 4
+	nuget Fable.Elmish.HMR ~> 7
+	nuget Fable.Remoting.Client ~> 7
+	nuget Feliz ~> 2
+	
+    //Build
+	nuget Fake.Core.Target ~> 5
+	nuget Fake.IO.FileSystem ~> 5
+	nuget Farmer ~> 1
+
+    //Testing
+	nuget Fable.Mocha ~> 2
+	nuget Expecto ~> 9
+	```
+	
+	- Run `dotnet paket install`
+	- Remove `Fable.React` and `Feliz.Bulma` from `src/Client/paket.references` (moved here according to @Freymaurer feedback)
+    - Run `dotnet paket update Fable.Remoting.Client`, needs to be done separately because version changes is a minor one and not major so I can't force it using only the `paket.dependencies` changes
+
+1. Remove all the webpack related dependencies:
```

Comment:
The engines need updating too (node -> 20, npm -> 10). Should probably happen before using npm.

Reply:
In TODOs above.